vulnerabilities-CVEd/CVE-2018-11759-Apache mod_jk access control bypass.bcheck

```diff
@@ -0,0 +1,25 @@
+metadata:
+    language: v2-beta
+    name: "CVE-2018-11759 - Apache mod_jk access control bypass"
+    description: "Checks for CVE-2018-11759 -Apache mod_jk access control bypass"
+    author: "CDonkin"
+    tags: "CVE-2018-11759", "mod_jk"
+
+run for each:
+    potential_path =
+        "/jkstatus",
+        "/jkstatus;"
+
+given host then
+    send request called check:
+        method: "GET"
+        path: {potential_path}
+
+    if "JK Status Manager" in {check.response.body} then
```

Comment:
You could consider checking the status code here as well so that you'll ignore any response body entirely when you get a 404.